src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -272,13 +272,9 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickResponseVisiblityCheckBoxForNewQuestion("RECEIVER_TEAM_MEMBERS");
         feedbackEditPage.clickVisibilityPreviewForNewQuestion();
         
-        feedbackEditPage.waitForElementVisibility(feedbackEditPage.getNewQnVisibilityMessage());
-        assertTrue("Expected recipient's team members to be able to see response, but was "
-                   + feedbackEditPage.getNewQnVisibilityMessage().getText(), 
-                   feedbackEditPage.getNewQnVisibilityMessage()
```

Comment:
Yup, it is used on line 281 and 282 of `InstructorFeedbackEditPageUiTest` to check that the visibility message does not contain some text after the recipient type has been changed